content/applications/hr/attendances/kiosks.rst

```diff
@@ -0,0 +1,187 @@
+======
+Kiosks
+======
+
+Odoo's *Attendances* app allows employees to check in and out of work directly from the database, or
+from a kiosk.
+
+Only employees with access to the database can check in and out from the database, and they are
+referred to as *users*. Kiosks are needed for employees who do **not** have access to the database.
+
+A kiosk is a dedicated device (a PC, tablet, or mobile phone) for employees to use when they check
+in and out.
+
+.. important::
+   If employees check in and out using a badge or an RFID, then an accessible device in *Kiosk Mode*
+   **must** be available in order to use these two methods.
+
+Configuration
+=============
+
+There are only a few configurations needed to use kiosks in the *Attendances* application. Navigate
+to :menuselection:`Attendances app --> Configuration` to access the :guilabel:`Settings` page.
```

Comment:
Add internal references to these configuration sections, to be clear on what needs to be configured
```suggestion
to :menuselection:`Attendances app --> Configuration` to access the :guilabel:`Settings` page to
configure the :ref:`attendances/kiosk-mode` and the :ref:`attendances/kiosk-settings`.
```